### PR TITLE
created a scaffolding for the project

### PR DESCRIPTION
**Pull request by tapioteras**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -0,0 +1,111 @@
+(defproject tribemon-client "0.1.0-SNAPSHOT"
+  :description "FIXME: write description"
+  :url "http://example.com/FIXME"
+  :license {:name "Eclipse Public License"
```
Comment (Verneri) [inline project.clj]: Why MIT? EPL is the standard for clojure

Comment (matiisi) [inline project.clj]: I prefer MIT over EPL since EPL is a wall of text that doesn't get read by anyone.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+target/
```
Comment (matiisi) [inline .gitignore]: add `figwheel_server.log`here as well.

Comment (tapioteras) [inline .gitignore]: done!

File: project.clj (hunk 1 above)
Comment (tapioteras) [inline project.clj]: done!

File: .gitignore (hunk 2 above)
Comment (matiisi) [inline .gitignore]: also remove the file from the previous commit :)

Comment (tapioteras) [inline .gitignore]: 😃  oh I forgot - removed!

**Pull request merged**